shared/actions/chat/index.js

```diff
@@ -156,10 +156,7 @@ function* _incomingMessage(action: Constants.IncomingMessage): SagaGenerator<any
         }
 
         if (pendingMessage) {
-          if (message.type === 'Attachment') {
-            // Copy locally-generated preview
-            message.previewPath = pendingMessage.previewPath
-          }
+          yield put(Creators.outboxMessageBecameReal(pendingMessage.key, message.key))
```

Comment:
This moves the local state that was indexed by outboxID over to the new key indexed by messageID.